flycheck.el

```diff
@@ -7051,6 +7052,17 @@ See URL `http://sass-lang.com'."
           line-end))
   :modes scss-mode)
 
+(flycheck-define-checker scss-lint
+  "A SCSS syntax checker using SCSS-Lint.
+
+See URL `https://github.com/causes/scss-lint'."
+  :command ("scss-lint" "--formatter default" source)
+  :error-patterns
+  ((error line-start (file-name) ":" line " [E] " (message) line-end)
+   (warning line-start (file-name) ":" line " [W] " (message) line-end))
+   :modes scss-mode
+   :next-checkers ((warning . scss)))
```

Comment:
Do you really want to apply the native SCSS compiler after `scss-lint`?  From your description I thought that SCSS-Lint is superior so that it makes the standard SCSS checker altogether redundant.